lib/releases.js

```diff
@@ -155,11 +156,18 @@ const generateChangeLog = (mergedPullRequests, config) => {
     categorizedPullRequests,
   ] = categorizePullRequests(mergedPullRequests, config)
 
+  const escapeTitle = (title) =>
+    // If config['change-title-escapes'] contains backticks, then they will be escaped along with content contained inside backticks
+    // If not, the entire backtick block is matched so that it will become a markdown code block without escaping any of its content
+    title.replace(new RegExp(`[${regexEscape(config['change-title-escapes'])}]|\`.*?\``, 'g'), (match) =>
+      match.length > 1 ? match
```

Comment:
Pass it to a function to make the conditional a bit more readable by using if statements?